### PR TITLE
Implement #260: Provide results summary per experiment

### PR DESCRIPTION
**Pull request by joschli**

### CODE REVIEW

**Comment by salsolatragus** [comment]
Experiment statistics, second table:

* [ ] There's a bug in experiment 2: Only one of the misuse is shown on the statics page
* [ ] For misuses with 0 findings, please show the thumbs down icon (detector missed the misuse)

**Comment by salsolatragus** [comment]
Fix test data

* [ ] Move no findings example to Ex1 or Ex3
* [ ] Ensure all detectors have the same misuses on all projects (with runs) in Ex1 and Ex3 (and not projects without misuses)

**Comment by joschli** [comment]
@salsolatragus I fixed the test data, this is ready to merge.

**Comment by salsolatragus** [comment]
I recreated the database and now Ex3 (and the full results statistics) fails with

    SQLSTATE[HY000]: General error: 1 no such table: detector_2 (SQL: select * from "detector_2" where "detector_2"."misuse_id" = 7 and "detector_2"."misuse_id" is not null order by "rank" asc)

File: mubench.reviewsite/src/Controllers/RunsController.php [hunk 1]
```diff
@@ -68,6 +68,45 @@ public function getResults(Request $request, Response $response, array $args)
...
+
+        $detectors = Detector::withRuns($experiment);
+        $results = $this->getResultsForExperiment($experiment, $detectors, $ex2_review_size);
+
+        $misuse_results = $this->getMisuseResults($experiment, $detectors);
+
+        return $this->renderer->render($response, 'experiment_stats.phtml', ['exp' => $experiment, 'experiment_results' => $results, 'ex2_review_size' => $ex2_review_size, 'misuses' => $misuse_results, 'det' => $detectors]);
+    }
+
+    public function getMisuseResults($experiment, $detectors)
```
Comment (salsolatragus) [inline mubench.reviewsite/src/Controllers/RunsController.php]: Test?

**Pull request merged**